corehq/apps/users/management/commands/populate_user_role.py

```diff
@@ -23,7 +22,7 @@ def sql_class(self):
 
     @classmethod
     def commit_adding_migration(cls):
-        return "TODO: add once the PR adding this file is merged"
+        return "4f5a5ef0a9b5ef9873a9b2dce5646d7aa881c416"
```

Comment:
I'd expected this to be a merge commit, but it's an update to staging.yaml. Should be fine as long as it's after the relevant PR was merged, just curious if that's intentional.

https://github.com/dimagi/commcare-hq/commit/4f5a5ef0a9b5ef9873a9b2dce5646d7aa881c416

Reply:
I just took the latest HEAD. As long as the commit has the migration in it I don't think it matters.